```
eng\targets\Cpp.Common.props(7,18): error MSB4066: The attribute "Include" in element <FilesToSign> is unrecognized.
 [D:\a\1\s\src\Servers\IIS\AspNetCoreModuleV1\AspNetCore\AspNetCore.vcxproj]
eng\targets\Cpp.Common.props(7,18): error MSB4066: The attribute "Include" in element <FilesToSign> is unrecognized.
 [D:\a\1\s\src\Servers\IIS\AspNetCoreModuleV2\AspNetCore\AspNetCore.vcxproj]
eng\targets\Cpp.Common.props(7,18): error MSB4066: The attribute "Include" in element <FilesToSign> is unrecognized.
 [D:\a\1\s\src\Servers\IIS\AspNetCoreModuleV2\CommonLibTests\CommonLibTests.vcxproj]
eng\targets\Cpp.Common.props(7,18): error MSB4066: The attribute "Include" in element <FilesToSign> is unrecognized.
 [D:\a\1\s\src\Servers\IIS\AspNetCoreModuleV1\IISLib\IISLib.vcxproj]
eng\targets\Cpp.Common.props(7,18): error MSB4066: The attribute "Include" in element <FilesToSign> is unrecognized.
 [D:\a\1\s\src\Servers\IIS\AspNetCoreModuleV2\CommonLib\CommonLib.vcxproj]
eng\targets\Cpp.Common.props(7,18): error MSB4066: The attribute "Include" in element <FilesToSign> is unrecognized.
 [D:\a\1\s\src\Servers\IIS\AspNetCoreModuleV2\gtest\gtest.vcxproj]
eng\targets\Cpp.Common.props(7,18): error MSB4066: The attribute "Include" in element <FilesToSign> is unrecognized.
 [D:\a\1\s\src\Servers\IIS\AspNetCoreModuleV2\IISLib\IISLib.vcxproj]
eng\targets\Cpp.Common.props(7,18): error MSB4066: The attribute "Include" in element <FilesToSign> is unrecognized.
 [D:\a\1\s\src\Servers\IIS\AspNetCoreModuleV2\InProcessRequestHandler\InProcessRequestHandler.vcxproj]
eng\targets\Cpp.Common.props(7,18): error MSB4066: The attribute "Include" in element <FilesToSign> is unrecognized.
 [D:\a\1\s\src\Servers\IIS\AspNetCoreModuleV2\OutOfProcessRequestHandler\OutOfProcessRequestHandler.vcxproj]
eng\targets\Cpp.Common.props(7,18): error MSB4066: The attribute "Include" in element <FilesToSign> is unrecognized.
 [D:\a\1\s\src\Servers\IIS\AspNetCoreModuleV2\RequestHandlerLib\RequestHandlerLib.vcxproj]
```

--- a/eng/targets/Cpp.Common.props
+++ b/eng/targets/Cpp.Common.props
@@ -4,8 +4,11 @@
   <PropertyGroup>
     <SignOutput Condition=" '$(SignType)' != '' ">true</SignOutput>
     <IsPackable>false</IsPackable>
-    <FilesToSign Include="$(MSBuildProjectDirectory)\$(OutDir)$(TargetName)$(TargetExt)" Authenticode="Microsoft400" />
   </PropertyGroup>
+
+  <ItemGroup>
+    <FilesToSign Include="$(MSBuildProjectDirectory)\$(OutDir)$(TargetName)$(TargetExt)" Authenticode="Microsoft400" />
+  </ItemGroup>
 
   <Import Project="MicroBuild.Plugin.props" Condition="'$(MicroBuildSentinelFile)' == ''" />
   <Import Project="$(MicroBuildPluginDirectory)\MicroBuild.Plugins.*\**\build\MicroBuild.Plugins.*.props" Condition=" '$(MicroBuildPluginDirectory)' != ''" />

